index.html

```diff
@@ -0,0 +1,135 @@
+       <div class="sep mx-3 my-4"></div>
+       <h6 class="venue text-primary mb-3">Dec 12th, 2022, 8pm | Zoom </h6>
+       <a href="#" class="learn py-2 px-2">Learn more</a>
+      </div>
+     <p class="theme h4 pt-5 px-2 text-center">Unite 100 000 Women in Tech to Foster Innovation with Purpose and Impact</p>
+
+    </main>
+  </section>
+<!-- End of Main Section -->
+<!-- Program Schedule -->
+<section class="program container-fluid">
+  <div class="program-box d-flex flex-column justify-content-center align-items-center">
+    <h2 class="h2 text-center pt-5">Summit Program</h2>
+    <div class="pt-1 underprogram"></div>
+  </div>
+  <div class=" d-md-flex flex-row mt-5 container">
+    <div class="program-item py-2 mx-1 mt-3">
+      <div class="program-logo"><i class="fa-sharp fa-solid fa-laptop-file"></i></div>
+      <div class="pro-topic"><h3>Works Exhibition</h3></div>
+      <div class="prosub-topic text-light text-center"><p>A number of Women and groups will have a chance to showcase their products and services in various booths.</p></div>
+    </div>
+    <div class="program-item py-2 mx-1 mt-3">
+      <div class="program-logo"><i class="fa-regular fa-lightbulb"></i></div>
+      <div class="pro-topic"><h3>Innovative session</h3></div>
+      <div class="prosub-topic text-light text-center"><p>The session will be a practical implementation of ideas that result in the introduction of new goods or services or improvement in offering goods or services.</p></div>
+    </div>
+    <div class="program-item py-2 mx-1 mt-3">
+      <div class="program-logo"><i class="fa-regular fa-gift"></i></div>
+      <div class="pro-topic"><h3>Award Presentation</h3></div>
+      <div class="prosub-topic text-light text-center"><p>Participants will get a chance to present their portfolio to sponsors. The winner with the best portfolio will also get $1000 cash price and instant employment</p></div>
+    </div>
+    <div class="program-item py-2 mx-1 mt-3 px-2">
+      <div class="program-logo"><i class="fa-regular fa-people-line"></i></div>
+      <div class="pro-topic"><h3>Interactive Lecture</h3></div>
+      <div class="prosub-topic text-light text-center"><p>Interactive session with leading women in Tech. The Lecture will be conducted in large groups.</p></div>
+    </div>
+  </div>
+  <div class="program-item py-2 px-2 mt-2 container">
+      <div class="program-logo"><i class="fa-solid fa-handshake"></i></div>
+      <div class="pro-topic"><h3>Networking</h3></div>
+      <div class="prosub-topic text-light text-center"><p>Participants will get a chance to connect and network with other women in different tech communities during this event. This will be a great opportunity build relations.</p></div>
+    </div>
+    <div class="program-bo d-flex flex-column justify-content-center align-items-center">
+      <h2 class="h4 text-center pt-2"><a href="#" class="text-dark">See more Programs</a></h2>
```

Comment:
- [ ] On the mobile version, this button changes styles. Please consider the following comparison 👇 
**Your design**
![image](https://user-images.githubusercontent.com/90222110/198262680-777a5616-92e8-45fb-8c7d-2e997bef049f.png)

**Given design**
![image](https://user-images.githubusercontent.com/90222110/198262759-dfbb68c7-dbc1-4c84-91f5-7a42ea8fea54.png)

Please add the current styles on the mobile screens to this button 😃 👍